RecoLocalCalo/HcalRecAlgos/plugins/HcalRecAlgoESProducer.cc

```diff
@@ -94,5 +96,19 @@ HcalRecAlgoESProducer::produce(const HcalSeverityLevelComputerRcd& iRecord)
    return myComputer ;
 }
 
+void HcalRecAlgoESProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+    edm::ParameterSetDescription desc;
+    desc.add<uint32_t>("phase",0);
+    desc.add<std::vector<std::string>>("RecoveredRecHitBits");
+    desc.add<std::vector<std::string>>("DropChannelStatusBits");
+    descriptions.add("hcalRecAlgos",desc);
+
+    edm::ParameterSetDescription desc_sevlvl;
+    desc_sevlvl.add<uint32_t>("Level");
+    desc_sevlvl.add<std::vector<std::string>>("RecHitFlags");
+    desc_sevlvl.add<std::vector<std::string>>("ChannelStatus");
+    desc_sevlvl.addVPSet("SeverityLevels",desc_sevlvl);
+}
```

Comment:
@Martin-Grunewald
yes, we consider now renaming fillDescription() and importing it under old regular name "SeverityLevel" into the regular config, so that thea latter will contain mostly 2017 era customization.